.vscode/configurationCache.log

```diff
@@ -0,0 +1 @@
inbarksdale/git/provider-jet-palette/.cache/tools/darwin_arm64/kubectl-v1.22.2","/Users/justinbarksdale/git/provider-jet-palette/.cache/tools/darwin_arm64/kustomize-v3.3.0","/Users/justinbarksdale/git/provider-jet-palette/.cache/tools/darwin_arm64/manifest-tool-v0.7.0","/Users/justinbarksdale/git/provider-jet-palette/.cache/tools/darwin_arm64/olm-bundle-v0.4.0","/Users/justinbarksdale/git/provider-jet-palette/.cache/tools/darwin_arm64/terraform-1.1.6","all","build","build.all","build.artifacts","build.artifacts.platform","build.check","build.check.platform","build.clean","build.code","build.code.platform","build.done","build.init","build.vars","check-diff","clean","cobertura","common.buildvars","config/schema.json","crds.clean","crossplane.help","debug.nuke","distclean","do.build.artifacts","do.build.artifacts.%","do.build.images","do.build.platform","do.build.platform.%","do.img.clean","e2e","e2e.done","e2e.init","e2e.run","fallthrough","fmt","fmt.simplify","generate","generate.done","generate.init","generate.run","go.build","go.cachedir","go.clean","go.distclean","go.fmt","go.fmt.simplify","go.generate","go.help","go.init","go.install","go.lint","go.test.codecov","go.test.integration","go.test.unit","go.validate","go.vendor","go.vendor.check","go.vendor.lite","go.vendor.update","go.vet","help","help-special","img.cache","img.clean","img.done","img.help","img.prune","img.release.build","img.release.build.crossplane.provider-jet-palette-controller.amd64","img.release.build.crossplane.provider-jet-palette-controller.arm64","img.release.build.crossplane.provider-jet-palette.amd64","img.release.build.crossplane.provider-jet-palette.arm64","img.release.clean","img.release.clean.crossplane.provider-jet-palette-controller.amd64","img.release.clean.crossplane.provider-jet-palette-controller.arm64","img.release.clean.crossplane.provider-jet-palette.amd64","img.release.clean.crossplane.provider-jet-palette.arm64","img.release.promote","img.release.promote.crossplane.provider-jet-palette-controller.amd64","img.release.promote.crossplane.provider-jet-palette-controller.arm64","img.release.promote.crossplane.provider-jet-palette.amd64","img.release.promote.crossplane.provider-jet-palette.arm64","img.release.publish","img.release.publish.crossplane.provider-jet-palette-controller.amd64","img.release.publish.crossplane.provider-jet-palette-controller.arm64","img.release.publish.crossplane.provider-jet-palette.amd64","img.release.publish.crossplane.provider-jet-palette.arm64","imports","imports.fix","k8s_tools.buildvars","lint","lint.done","lint.init","lint.run","output.clean","output.init","promote","promote.artifacts","promote.init","prune","publish","publish.artifacts","publish.init","release.tag","reviewable","run","submodules","tag","test","test.done","test.init","test.run","vendor","vendor.check","vendor.update","version.isdirty","vet"],"launchTargets":[],"customConfigurationProvider":{"workspaceBrowse":{"browsePath":[],"compilerArgs":[]},"fileIndex":[]}}
```

Comment:
do you want to remove/ignore this entire .vscode folder from source control? We've typically added an entry in the .gitignore for these types of files that are specific to the individual developer as opposed to the entire community.  See https://github.com/crossplane/crossplane/blob/master/.gitignore#L19 as an example :)